alacritty_terminal/src/term/mod.rs

```diff
@@ -706,14 +712,26 @@ impl VisualBell {
                 // Since we want the `intensity` of the VisualBell to decay over
                 // `time`, we subtract the `inverse_intensity` from 1.0.
                 1.0 - inverse_intensity
-            },
+            }
+        }
+    }
+
+    pub fn run_command(&self) {
+        if let Some(ref cmd) = self.command {
+            match start_daemon(cmd.program(), cmd.args()) {
+                Ok(_) => debug!("Launched {} with args {:?} on bell", cmd.program(), cmd.args()),
+                Err(_) => {
+                    warn!("Unable to launch {} with args {:?} on bell", cmd.program(), cmd.args())
+                }
+            }
         }
     }
 
     pub fn update_config(&mut self, config: &Config) {
-        let visual_bell_config = &config.visual_bell;
-        self.animation = visual_bell_config.animation;
-        self.duration = visual_bell_config.duration();
+        let bell_config = &config.bell;
```

Comment:
Defining this variable here has no benefits, since `config.bell` and `bell_config` are exactly the same length.